Content/Library/docsTool/templates/types.fs

```diff
@@ -0,0 +1,127 @@
+module Types
+
+open System
+open Fable.React
+open Fable.React.Props
+open FSharp.MetadataFormat
+open PartMembers
+
+
+let generateTypeDocs (model : TypeInfo) (props) =
+    let members = model.Type.AllMembers
+    let comment = model.Type.Comment
+    let ``type`` = model.Type
+    let byCategory =
+        members
+        |> List.groupBy (fun m -> m.Category)
+        |> List.sortBy (fun (k,v) -> if String.IsNullOrEmpty(k) then "ZZZ" else k )
+        |> List.mapi (fun i (k,v) -> {
+            Index = i
+            GroupKey = k
+            Members = v |> List.sortBy (fun m -> if m.Kind = MemberKind.StaticParameter then "" else m.Name)
+            Name = if String.IsNullOrEmpty(k) then "Other type members" else k
+        })
+    [
+        yield h1 [] [
+            str model.Type.Name
+        ]
+
+        yield p [] [
+            if model.Type.IsObsolete then
+                yield div [
+                    Class "alert alert-warning"
+                ] [
+                    strong [] [
+                        str "WARNING: "
+                    ]
+
+                    str " This API is obsolete"
+                ]
+
+            yield span [] [
+                str (sprintf "Namespace: %s" model.Namespace.Name)
+            ]
+
+            yield br []
+
+            if model.HasParentModule then
+                yield span [] [
+                    str "Parent Module: "
+
+                    a [
+                        Href (sprintf "%s.html" model.ParentModule.Value.UrlName)
+                    ] [
+                        str model.ParentModule.Value.Name
+                    ]
+                ]
+
+                yield br []
+
+            if ``type``.Attributes |> Seq.isEmpty |> not then
+                yield span [] [
+                    yield str "Attributes: "
+
+                    yield br []
+
+                    for attr in ``type``.Attributes do
+                        yield str (attr.Format())
+                        yield br []
+                ]
+        ]
+
+        yield div [
+            Class "xmldoc"
+        ] [
+            for sec in comment.Sections do
+                if byCategory |> Seq.exists (fun m -> m.GroupKey = sec.Key) |> not then
```

Comment:
similarly, I've seen this xmldoc section repeated as well. seems like a good place for a component/function.